templates/contribute.html

```diff
@@ -0,0 +1,140 @@
+{% extends 'global/base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+
+    <div class="container">
+
+        <section id="resources">
+            <div class="row clearfix">
+                <div class="col-md-12">
+                    <h2>Contribute</h2>
+                    <p>
+                        There are many ways to contribute to Django Girls: you can organize an event, coach, work on the tutorial and help on Gitter. If you'd rather help us financially, you can always support us on <a href="https://www.patreon.com/djangogirls">Patreon</a>.
+                    </p>
+                </div>
+            </div>
+
+            <div class="row clearfix">
+                <div class="col-md-6">
+                    <h3>Support us!</h3>
+                    <div class="cover">
+                        <a href="https://www.patreon.com/djangogirls">
+                            <img src="{% static 'img/global/contribute/love.jpg' %}" alt="Pixel heart and Django." />
+                        </a>
+                    </div>
+                    <div class="info">
+
+                        <p>Django Girls is a <a href="http://apps.charitycommission.gov.uk/Showcharity/RegisterOfCharities/CharityFramework.aspx?RegisteredCharityNumber=1163260&SubsidiaryNumber=0">Registered Charity</a> and is volunteers run. If you're an individual or a company and want to support us, you can make a <a href="https://www.patreon.com/djangogirls">monthly donations</a>. For a one time donation, you can send it to our <a href="mailto:hello@djangogirls.org">Paypal account</a>.</p>
```

Comment:
"and is volunteer run." Volunteer should be singular. 

Also, you can "make a monthly donation." Again, "donation" should be singular. 

Edit to "You can send one-time donations to our PayPal account." (Note capitalization of PayPal.)